lib/src/speed_dial.dart

```diff
@@ -217,23 +221,44 @@ class _SpeedDialState extends State<SpeedDial> with SingleTickerProviderStateMix
       curve: widget.curve,
     );
 
-    return Positioned(
-      bottom: widget.marginBottom - 16,
-      right: widget.marginRight - 16,
-      child: Container(
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.end,
-          crossAxisAlignment: CrossAxisAlignment.end,
-          children: List.from(fabChildren)
-            ..add(
-              Container(
-                margin: EdgeInsets.only(top: 8.0, right: 2.0),
-                child: animatedFloatingButton,
-              ),
+    switch (widget.orientation) {
+      case SpeedDialOrientation.Up:
+        return Positioned(
+          bottom: widget.marginBottom - 16,
+          right: widget.marginRight - 16,
+          child: Container(
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.end,
+              crossAxisAlignment: CrossAxisAlignment.end,
+              children: List.from(fabChildren)
+                ..add(Container(
+                  margin: EdgeInsets.only(top: 8.0, right: 2.0),
+                  child: animatedFloatingButton,
+                )),
             ),
-        ),
-      ),
-    );
+          ),
+        );
+        break;
+      case SpeedDialOrientation.Down:
+        return Positioned(
+          top: MediaQuery.of(context).size.height - 56 - 8 - (widget.marginBottom - 16),
```

Comment:
I understand why you put those 8px here, but this causes the main button and the first child to be too close from each other. The solution is to give every child a bottom margin instead of a top one ( see next comment ).

Reply:
Done.